tools/ci/production-helper

```diff
@@ -50,6 +50,7 @@ fi
 # TODO: Ideally this would test actually logging in, but this is a start.
 echo; echo "Now testing that the newly installed server's homepage loads"; echo
 
+cp -a tools/ci/success-http-headers.txt ~/
```

Comment:
OK, does that mean we can delete a similar copy step from `setup-production`, so it's a move?